code/modules/surgery/organs/augments_internal.dm

```diff
@@ -140,9 +140,9 @@
 	..()
 	if(crit_fail)
 		return
-	if(owner.stunned > stun_max_amount)
+	if(owner.AmountStun() > stun_max_amount)
 		owner.SetStunned(stun_max_amount)
-	if(owner.weakened > stun_max_amount)
+	if(owner.AmountWeakened() > stun_max_amount)
```

Comment:
Make sure stun_max_amount is updated as well to address for the fact that stuns are now *20